models/Item.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require("mongoose");
+
+const itemSchema = new mongoose.Schema({
+    // var name below TBC
+    postID: {
+        type: Number,
+        required: true,
+    },
+    type: {
+        type: ["Request", "Offer"],
+        required: true,
+    },
```

Comment:
type should be String. to limit possible options, use enum. also maybe rename to postType so as not to confuse with type?
```
postType: {
  type: String,
  enum: ["request", "offer"],
  required: true
}
```